build.gradle

```diff
@@ -10,16 +10,13 @@ version = "5.5.0-SNAPSHOT"
 
 repositories {
     mavenLocal()
-    mavenCentral()
     jcenter()
 }
 
 dependencies {
-    implementation "org.openmicroscopy:omero-blitz:5.5.0-SNAPSHOT"
-    implementation "commons-beanutils:commons-beanutils:1.9.3"
-    implementation "com.zeroc:icegrid:3.6.4"
-    implementation "commons-lang:commons-lang:2.5"
-    implementation "commons-collections:commons-collections:3.2"
-    testCompile "org.testng:testng:6.9.8"
+    api("org.openmicroscopy:omero-blitz:5.5.0-m4")
+    implementation("commons-beanutils:commons-beanutils:1.9.3")
+    implementation("com.zeroc:icegrid:3.6.4")
```

Comment:
I'm surprised this is necessary. Does `omero-blitz` not expose this? cc: @jburel @rgozim

Reply:
it should be exposed

Reply:
Actually they are not.
Without those 2 dependencies it does not build